### PR TITLE
Better ORM and vendor icons, reworked Resonators

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

This PR resprites ORM and Mining Vendor(Credits to https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9678 for the icons), changes mining vendor voucher reedem menu to radial one, and reworks resonators. They are no longer completely useless. Now they shoot 1 resonating holoball per second(per 0.8 for advanced one), which deals 5 damage(harmless to mobs) and creates a resonator field that detonates after 1 second. You can also manually detonate all fields by using resonator in-hand. Alt-clicking the resonator changes the rupture time of the traps(1 second or 5 seconds, second mode is useful for setting up traps for fauna).

## Why It's Good For The Game

New icons fit bee style nicely, while resonators are worse than picks rn

## Changelog
:cl:
tweak: Reworked resonators
imageadd: Added new icons for ORM and mining vendor
/:cl:

### CODE REVIEW

**Comment by february-ops** [comment]
Radial bad

**Comment by SmArtKar** [comment]
> Radial bad

Radial good. It doesnt make the whole vendor work on radial system, just voucher reedeming

**Comment by Crossedfall** [comment]
No attributions provided. In order for this to be considered you need to link to the PRs you're porting from and/or provide cherry-picked commits adding the original authors in as co-authors. See https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors for more information on adding co-authors to commits. 

You are welcome to re-open this after providing the appropriate credits.

**Comment by SmArtKar** [comment]
https://github.com/Citadel-Station-13/Citadel-Station-13/commit/9f97f768da2021c6d8fdf2a4edc09796d9b21860 Here's the sprites.

**Comment by Crossedfall** [comment]
Can you please add a co-author commit attributing them? See the above link on how to do it, the email (unless otherwise listed) is going to be `username@users.noreply.github.com` (where `username` should be the person who added them)

**Comment by PowerfulBacon** [comment]
The ORM is ugly and has a bad perspective in my opinion, also the bs miner and stacker are swapped for no reason

**Comment by SmArtKar** [comment]
> The ORM is ugly and has a bad perspective in my opinion, also the bs miner and stacker are swapped for no reason

Gonna change ORM a bit. Ore stacker was white while other mining machinery was grey. It was really standing out.

File: code/modules/mining/equipment/resonator.dm [hunk 1]
```diff
@@ -10,60 +10,91 @@
...
-	if(check_allowed_items(target, 1))
-		CreateResonance(target, user)
-	return TRUE
+/obj/item/resonator/AltClick(mob/user)
+	to_chat(user, "<span class='info'>You switched [src] to [trap_mode ? "mining":"trapping"] mode.</span>")
+	trap_mode = !trap_mode
+	. = ..()
+
+/obj/item/resonator/proc/Recharge()
+	charged = TRUE
```
Comment (PowerfulBacon) [inline code/modules/mining/equipment/resonator.dm:62]: Should have a cool sound for it

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Wilchenx** [comment]
Maybe this should be atomized?

**Comment by St0rmC4st3r** [comment]
Definitely should be atomized. The resonator changes are very good, while the sprites sucks ass like the majority of new sprites on /tg/ or citadel, since the perspective is inconsistent and ugly.

**Comment by theboy6545** [comment]
Please atomise this, I like the changes but the sprites are ugly as all hell, and I'm pretty sure azlan is making new sprites soon anyways.

**Pull request closed without merging**